src/redux/selectors/index.js

```diff
@@ -209,3 +209,12 @@ export const patientAgeAtResourcesSelector = createSelector(
     });
   },
 );
+
+export const orderedResourceTypeFiltersSelector = createSelector(
+  [resourceTypeFiltersSelector],
+  (resourceTypeFilters) => Object.keys(resourceTypeFilters).sort()
```

Comment:
should the keys be sorted, or the labels?  
...this could get flaky, down the road, if there is a more significant mismatch between the `type` key, and its label.

Reply:
I can update this to sort by label

Reply:
Actually, this is probably longer than a 5 minute refactor. We'll need to do this after the demo.